### PR TITLE
amp 0.6.2 (new formula)

### PR DESCRIPTION
**Pull request by vladimyr**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

**Comment by vladimyr** [comment]
@chenrui333 Tests are now passing :tada:

File: Formula/amp.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+class Amp < Formula
+  desc "Text editor for your terminal"
+  homepage "https://amp.rs"
+  url "https://github.com/jmacdonald/amp/archive/0.6.2.tar.gz"
+  sha256 "9279efcecdb743b8987fbedf281f569d84eaf42a0eee556c3447f3dc9c9dfe3b"
+  head "https://github.com/jmacdonald/amp.git"
+
+  depends_on "cmake" => :build
```
Comment (chenrui333) [inline Formula/amp.rb:8]: I thought you only used cargo for the build, so why do we need this cmake dep?

Comment (vladimyr) [reply Formula/amp.rb:8]: Because the dependency graph contains some other dependency that apparently uses it for compilation 🤷

Comment (chenrui333) [reply Formula/amp.rb:8]: yeah, amp is using cmake underlyingly.

```
Last 15 lines from /Users/rchen/Library/Logs/Homebrew/amp/01.cargo:
running: "cmake" "/Users/rchen/Library/Caches/Homebrew/cargo_cache/registry/src/github.com-1ecc6299db9ec823/onig_sys-68.0.1/oniguruma" "-DBUILD_SHARED_LIBS=OFF" "-DCMAKE_INSTALL_PREFIX=/private/tmp/amp-20200621-83707-856n8t/amp-0.6.2/target/release/build/onig_sys-6385270c2b3e2efc/out" "-DCMAKE_C_FLAGS= -ffunction-sections -fdata-sections -fPIC --target=x86_64-apple-darwin" "-DCMAKE_C_COMPILER=/usr/local/Homebrew/Library/Homebrew/shims/mac/super/clang" "-DCMAKE_CXX_FLAGS= -ffunction-sections -fdata-sections -fPIC --target=x86_64-apple-darwin" "-DCMAKE_CXX_COMPILER=/usr/local/Homebrew/Library/Homebrew/shims/mac/super/clang++" "-DCMAKE_BUILD_TYPE=Release"
```

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@vladimyr, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**